### DISCUSSION TITLE
Capture view on react native svg

### DISCUSSION THREAD

**Discussion by taduyde** [software-mansion/react-native-svg#1828] [General]
I'm using this plugin(https://github.com/gre/react-native-view-shot) to capture the view, but it isn't working with react native SVG
How can I fix the plugin?

**Comment by bohdanprog** [comment]
Hello @taduyde,
Can you provide an example, please?
Thank you

**Comment by jakex7** [comment]
[react-native-view-shot](https://github.com/gre/react-native-view-shot) does work with react-native-svg
